src/main/java/se/sundsvall/document/api/model/DocumentCreateRequest.java

```diff
@@ -0,0 +1,88 @@
+package se.sundsvall.document.api.model;
+
+import static io.swagger.v3.oas.annotations.media.Schema.RequiredMode.REQUIRED;
+
+import java.util.List;
+import java.util.Objects;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.NotBlank;
+import se.sundsvall.dept44.common.validators.annotation.ValidMunicipalityId;
+
+@Schema(description = "DocumentCreateRequest model.")
+public class DocumentCreateRequest {
+
+	@ValidMunicipalityId
+	@Schema(description = "Municipality ID", example = "2281", requiredMode = REQUIRED)
+	private String municipalityId;
+
+	@NotBlank
+	@Schema(description = "Actor that created this revision.", example = "username123", requiredMode = REQUIRED)
+	private String createdBy;
+
+	@Schema(description = "List of DocumentMetadata objects.")
```

Comment:
Don't know the domain that well, but does it make sense to create a document without metadata? I.e. should there be a `@NotEmpty` or similar here?

Reply:
You are right! Fixed now.